ceph-deploy-tag/build/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+if [ "$TAG" = false ] ; then
+    echo "Assuming tagging process has succeeded before because TAG was set to false"
+    exit 0
+fi
+
+# Create the virtualenv
+virtualenv venv
```

Comment:
We should use the new scripts in scripts/build_utils.sh to make the virtualenv and install ansible